Vagrantfile

```diff
@@ -155,50 +155,37 @@ SCRIPT
 # To support VirtualBox linked clones
 Vagrant.require_version(">= 1.8")
 
-Vagrant.configure("2") do |config|
-  config.vm.box = "centos/7"
-  config.vm.box_version = "1811.02"
-
-  config.vm.provider "virtualbox" do |v|
-    v.linked_clone = true
-    v.memory = 2048
-    v.cpus = 2
-    v.customize ["modifyvm", :id, "--chipset", "ich9"]
+def declare_bootstrap(machine, os_data)
+  machine.vm.box = os_data[:name]
+  machine.vm.box_version = os_data[:version]
+
+  machine.vm.hostname = "bootstrap"
+  machine.vm.provider "virtualbox" do |v|
+    v.memory = 4086
+    machine.vm.synced_folder ".", "/vagrant", type: "virtualbox"
   end
 
-  config.vm.network "private_network",
-    type: :dhcp,
-    nic_type: 'virtio',
-    **CONTROL_PLANE_NETWORK
-  config.vm.network "private_network",
-    type: :dhcp,
-    nic_type: 'virtio',
-    **WORKLOAD_PLANE_NETWORK
+  machine.vm.provision "import-release",
+    type: "shell",
+    inline: IMPORT_RELEASE
 
-  config.vm.define :bootstrap, primary: true do |bootstrap|
-    bootstrap.vm.hostname = "bootstrap"
+  machine.vm.provision "import-ssh-private-key",
+    type: "shell",
+    inline: IMPORT_SSH_PRIVATE_KEY
 
-    bootstrap.vm.provider "virtualbox" do |v|
-      v.memory = 4096
-      bootstrap.vm.synced_folder ".", "/vagrant", type: "virtualbox"
-    end
+  machine.vm.provision "bootstrap",
+    type: "shell",
+    inline: BOOTSTRAP
 
-    bootstrap.vm.provision "import-release",
-      type: "shell",
-      inline: IMPORT_RELEASE
+  bootstrap.vm.provision "create-volumes",
+    type: "shell",
+    inline: CREATE_VOLUMES
 
-    bootstrap.vm.provision "import-ssh-private-key",
-      type: "shell",
-      inline: IMPORT_SSH_PRIVATE_KEY
 
-    bootstrap.vm.provision "bootstrap",
-      type: "shell",
-      inline: BOOTSTRAP
+  return machine
```

Comment:
`machine` is enough, explicit return are not idiomatic in Ruby: https://github.com/rubocop-hq/ruby-style-guide#no-explicit-return